app/src/main/java/chat/rocket/android/main/ui/MainActivity.kt

```diff
@@ -268,4 +269,14 @@ class MainActivity : AppCompatActivity(), MainView, HasActivityInjector, HasSupp
             }
         }
     }
+
+    override fun onSaveInstanceState(outState: Bundle?) {
+        super.onSaveInstanceState(outState)
+        outState?.putBoolean(CURRENT_STATE,isFragmentAdded)
```

Comment:
There is only a need of adding a space after `,` here but we can fix it later.
Thanks for your PR @divyanshub024 !

Reply:
ok, I'll fix that in next PR.